src/js/services/user-service.js

```diff
@@ -17,7 +17,9 @@ export const EDITABLE_PROPERTIES = [
     'surname',
     'email',
     'signedUpAt',
-    'properties'
+    'properties',
+    'pendingClients',
```

Comment:
true. I had added this back then when I was using `immediateUpdate`. will remove